pkg/promclient/promclient.go

```diff
@@ -46,6 +48,8 @@ var (
 		http.StatusServiceUnavailable:  codes.Unavailable,
 		http.StatusInternalServerError: codes.Internal,
 	}
+
+	ThanosUserAgent = fmt.Sprintf("Thanos/%s", version.Version)
```

Comment:
`pkg/http/http.go` has the same exact user-agent string. Maybe it would be worth reusing that?

Reply:
good point